druntime/src/core/demangle.d

```diff
@@ -2901,3 +2911,76 @@ private string toStringConsume (immutable ManglingFlagInfo[] infos, ref ushort b
     }
     return null;
 }
+
+private shared CXX_DEMANGLER __cxa_demangle;
+
+/**
+ * Returns:
+ *  a CXX_DEMANGLER if a C++ stdlib is loaded
+ */
+
+CXX_DEMANGLER getCXXDemangler() nothrow @trusted
+{
+    if (__cxa_demangle is null)
+    version (Posix)
+    {
+        import core.sys.posix.dlfcn : dlsym;
+        version (DragonFlyBSD) import core.sys.dragonflybsd.dlfcn : RTLD_DEFAULT;
+        version (FreeBSD) import core.sys.freebsd.dlfcn : RTLD_DEFAULT;
+        version (linux) import core.sys.linux.dlfcn : RTLD_DEFAULT;
+        version (NetBSD) import core.sys.netbsd.dlfcn : RTLD_DEFAULT;
+        version (OSX) import core.sys.darwin.dlfcn : RTLD_DEFAULT;
+        version (Solaris) import core.sys.solaris.dlfcn : RTLD_DEFAULT;
+
+        if (auto found = cast(CXX_DEMANGLER) dlsym(RTLD_DEFAULT, "__cxa_demangle"))
```

Comment:
Shouldn't this be from the `__cxxabiv1` namespace?

Reply:
That is what I thought as first, but in my `libstdc++` it is not in a namespace. And apparently, it works on all CI targets as well.

```
omer@omer-ThinkPad-E14-Gen-3:/lib/x86_64-linux-gnu$ readelf -sW libstdc++.so.6.0.30 | grep demangle
  4302: 00000000000ba1c0   371 FUNC    GLOBAL DEFAULT   15 __cxa_demangle@@CXXABI_1.3
```